lua/lspconfig/server_configurations/rust_analyzer.lua

```diff
@@ -1,13 +1,16 @@
 local util = require 'lspconfig.util'
 
-local function reload_workspace(bufnr)
-  bufnr = util.validate_bufnr(bufnr)
-  vim.lsp.buf_request(bufnr, 'rust-analyzer/reloadWorkspace', nil, function(err)
-    if err then
-      error(tostring(err))
-    end
-    vim.notify 'Cargo workspace reloaded'
-  end)
+local function reload_workspace()
```

Comment:
this function still need bufnr. then you can use it to get the client which attach this buffer .and send  request.